src/checkers/inference/InferenceAnnotatedTypeFactory.java

```diff
@@ -166,6 +166,14 @@ public CFTransfer createFlowTransferFunction(CFAbstractAnalysis<CFValue, CFStore
 
     @Override
     public TreeAnnotator createTreeAnnotator() {
+        if (realTypeFactory instanceof InferrableAnnotatedTypeFactory) {
+            TreeAnnotator realInferenceTreeAnnotator = ((InferrableAnnotatedTypeFactory) realTypeFactory)
+                    .getInferenceTreeAnnotator(this, realChecker,
+                            variableAnnotator, slotManager);
+            return new ListTreeAnnotator(new ImplicitsTreeAnnotator(this),
```

Comment:
In my previous comment about this method I meant that here you should just return realInferenceTreeAnnotator.
Maybe some future subclass doesn't want the ImplicitsTreeAnnotator; maybe another wants it in a different order.
If you implement InferrableATF, it's up to you to return a ListTreeAnnotator with all required parts.